stake-pool/cli/src/main.rs

```diff
@@ -1158,14 +1161,55 @@ fn command_update(
 
     let validator_list = get_validator_list(&config.rpc_client, &stake_pool.validator_list)?;
 
-    let (mut update_list_instructions, final_instructions) =
+    let (mut update_list_instructions, final_instructions) = if force {
         spl_stake_pool::instruction::update_stake_pool(
             &spl_stake_pool::id(),
             &stake_pool,
             &validator_list,
             stake_pool_address,
             no_merge,
-        );
+        )
+    } else {
+        let (withdraw_authority, _) =
+            find_withdraw_authority_program_address(&spl_stake_pool::id(), stake_pool_address);
+        let update_list_instructions: Vec<Instruction> = validator_list
+            .validators
+            .chunks(MAX_VALIDATORS_TO_UPDATE)
+            .enumerate()
+            .filter_map(|(i, chunk)| {
```

Comment:
Should we work this into a function in the program `instruction.rs` instead, alongside `update_stake_pool`? Then the functionality could be accessed from more than just the CLI.

Reply:
Yes, i was thinking of doing the same. How's `update_stale_stake_pool` for the fn name?

Reply:
Yep, that was what I had in mind, too!